cmd/kspn/main.go

```diff
@@ -63,38 +55,25 @@ func init() {
 		nodecmd.AccountCommand,
 
 		// See utils/nodecmd/consolecmd.go:
-		nodecmd.GetConsoleCommand(nodeFlags, rpcFlags),
+		nodecmd.GetConsoleCommand(nodecmd.KspnNodeFlags(), nodecmd.CommonRPCFlags),
 		nodecmd.AttachCommand,
 
 		// See utils/nodecmd/versioncmd.go:
 		nodecmd.VersionCommand,
 
 		// See utils/nodecmd/dumpconfigcmd.go:
-		nodecmd.GetDumpConfigCommand(nodeFlags, rpcFlags),
+		nodecmd.GetDumpConfigCommand(nodecmd.KspnNodeFlags(), nodecmd.CommonRPCFlags),
 	}
 	sort.Sort(cli.CommandsByName(app.Commands))
 
-	app.Flags = append(app.Flags, nodeFlags...)
-	app.Flags = append(app.Flags, rpcFlags...)
-	app.Flags = append(app.Flags, nodecmd.ConsoleFlags...)
-	app.Flags = append(app.Flags, debug.Flags...)
+	app.Flags = nodecmd.KspnAppFlags()
 
 	cli.AppHelpTemplate = utils.GlobalAppHelpTemplate
 	cli.HelpPrinter = utils.NewHelpPrinter(utils.CategorizeFlags(app.Flags))
 
 	app.CommandNotFound = nodecmd.CommandNotExist
 
-	app.Before = func(ctx *cli.Context) error {
-		runtime.GOMAXPROCS(runtime.NumCPU())
-		logDir := (&node.Config{DataDir: utils.MakeDataDir(ctx)}).ResolvePath("logs")
-		debug.CreateLogDir(logDir)
-		if err := debug.Setup(ctx); err != nil {
-			return err
-		}
-		metricutils.StartMetricCollectionAndExport(ctx)
-		utils.SetupNetwork(ctx)
-		return nil
-	}
+	app.Before = nodecmd.BeforeRunNode
```

Comment:
Same question as the above
https://github.com/klaytn/klaytn/pull/1580/files#r944160746